src/native/macos.rs

```diff
@@ -901,6 +901,40 @@ unsafe fn set_icon(ns_app: ObjcId, icon: &Icon) {
     CGImageRelease(image);
 }
 
+/// Initialize the system menu bar for this application
+/// - ns_app: This NSApplication
+#[no_mangle]
```

Comment:
does it really need to be `no_mangle`?

Reply:
Definitely not. I was messing with the debugger and forgot to remove that.